css/style.css

```
@@ -16,4 +16,326 @@ body {
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.click {
+    border-radius: 3px;
+    font-size: 12px;
+    background-color: #ff6200;
+    border-color: #ff6200;
+    width: 30%;
+    margin: 20px 0px 20px 0px;
+    padding: 15px 5px 15px 5px;
+    color: white;
+}
+
+#subheading {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 40px;
+}
+
+#subheading h2 {
+    font-size: 24px;
+    font-weight: normal;
+}
+
+.features {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.feature-container {
+    display: flex;
+    flex-direction: column;
+    height: 140px;
+    width: 250px;
+    margin: 0;
+    padding: 0;
+}
+
+.feature-container img {
+    width: 200px;
+    margin: auto;
+    padding-bottom: 0;
+}
+
+.features h4 {
+    font-size: 14px;
+    font-weight: 540;
+    margin: auto;
+    padding-top: 0;
+}
+
+#extension {
+    display: flex;
+    flex-direction: row;
+    padding: 20px 0 20px 0;
+    margin: 20px 0 20px 0;
+}
+
+#ext-container {
+    flex-basis: 40%;
+}
+
+#ext-img {
+    padding-left: 0;
+    margin-left: 0;
+}
+
+#sales-container {
+    flex-basis: 60%;
+    background-color: #fef9f3;
+    display: flex;
+    flex-direction: column;
+    vertical-align: middle;
+}
+
+#sales-container h3 {
+    font-style: italic;
+    font-size: 24px;
+    font-weight: normal;
+    text-align: center;
+    align-content: baseline;
+    width: 60%;
+    margin: 60px 150px;
+    margin-bottom: 0;
+}
+
+.clicky {
+    border-radius: 5px;
+    font-size: 16px;
+    background-color: #ff6200;
+    border-color: #ff6200;
+    width: 30%;
+    margin: 30px 260px;
+    padding: 15px 5px 15px 5px;
+    color: white;
+}
+
+hr {
+    border: 0;
+    height: 1px;
+    width: 400px;
+    background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0 ,0));
+}
+
+#footer-container {
+    display: flex;
+    flex-direction: column;
+} 
+
+#contact {
+    text-align: center;
+    font-size: 12px;
+    font-weight: 400;
+    color: #606060;
+}
+
+#social-media {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+#icon {
+    margin: 0.2em;
+    border: 3px solid #F5F5F5;
+    padding: 6px;
+    border-radius: 80%;
+    width: 10px;
+    height: 10px; 
+}
+
+.copyright {
+    text-align: center;
+    font-size: 12px;
+    font-weight: 400; 
+    color: lightgrey;  
+    padding: 15px;
+}
+
+@media only screen and (max-width: 600px) {
+    
+    #header {
+        max-height: 28px;
+    }
+
+    #header-img {
+        padding: 5px 7px 2px 60px;
+        width: 20px;
+        height: 20px;
+        position: relative;
+    }
+    
+    #nav-bar {
+        padding: 7px 5px 5px 60px;
+    }
+    
+    a {
+        font-size: 15px;
+        padding: 12px 10px;
+    }
+    
+    #background-image {
+        background-repeat: no-repeat;
+        margin-top: 5px;
+        display: block;
+        width: 100%;
```

```suggestion
        background-repeat: no-repeat;
        margin-top: 5px;
        display: block;
        width: 100%;
```
What happens if you comment these? and changes?